src/components/DeliverableView/Metadata.tsx

```diff
@@ -24,14 +24,18 @@ const Metadata = (props: ViewProps): JSX.Element => {
         >
           <InternalComment deliverable={deliverable} />
 
-          {deliverable.status !== 'Rejected' && <DeliverableStatusBadge status={deliverable.status} />}
+          <div style={{ float: 'right', marginBottom: '0px', marginLeft: '16px' }}>
```

Comment:
Move this inside the conditional? Otherwise we might have 2 divs with a marginLeft, maybe some extra space there which might show depending on the description content.

Reply:
🤦‍♂️